### PR TITLE
Schedule meetups through November 2018

### PR DESCRIPTION
**Pull request by dasanchez**
FreeGeek and HackLab will host the following events:
- Aug 23: Meet and Greet @ FreeGeek
- Sep 20: Meet and Greet @ HackLab
- Oct 25: Meet and Greet @ FreeGeek
- Nov 22: Planning Session @ HackLab

### CODE REVIEW

**Comment by Shrinks99** [comment]
Should the planning meeting specify that it is mainly aimed at current members?  While I know it's always the last one up and then we don't plan anything else until it's finished I think it would be good to direct new people to the meet & greet sessions instead.

**Comment by dasanchez** [comment]
@Shrinks99 I would like it if people attending the planning session were aware of recent activities and had some context to contribute and feel like they belong. If a planning session is your first time attending a meetup, you might be a bit lost and require some explanation.

That said, I like the idea of being inclusive and not having an "invite-only" club. I'm OK with rewording the last statement to suggest newcomers attend a monthly meetup first:

> New to Toronto Mesh? Join the chat or visit a monthly meetup to meet the group and become familiar with ongoing activities!

**Comment by Shrinks99** [comment]
I like rewording the last statement to make that more clear.  

While I don't want Toronto Mesh to be an invite only club by any means I wouldn't be necessarily opposed to restricting planning meetings to existing folks (those who are on the chat and do things?  We don't need to have anything super strict like "you can't come here" style or any nonsense like that) as I don't think the planning meetings are a particularly great introduction for the users that find our website on the internet, see that there's a meeting coming up and go to it to meet people and see whats up.  They're an hour long and full of information that is useful to pretty much nobody who is new and just wants to learn about what mesh networking is.

Of course this needs to be brought up with more people, maybe it's something to address at the planning meeting? :P

**Comment by dasanchez** [comment]
Thanks @Shrinks99! I updated the closing message for the planning session.

File: _posts/2018-11-22-planning-session.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+endTime: '21:30'
+---
+
+This us Toronto Mesh's seventh planning session. We will regroup and plan for the winter and beyond! During this session we will:
+- Review all projects and efforts undertaken since April.
+- Re-evaluate our long-term goals.
+- Set the priorities and projects to work on for the next six months.
+- Assign working group leads for the next six months. You can see our current [working groups and governance processes here](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md)!
+
+
```
Comment (benhylau) [inline _posts/2018-11-22-planning-session.md]: Random newline?

File: _posts/2018-10-25-meet-n-greet.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+layout: post
+title: "Meet and Greet"
+slug: meet-and-greet
+text: 'Meet the members of Toronto Mesh'
+location: Free Geek Toronto, 180 Sudbury St
+locationLink: https://osm.org/go/ZX6BI0AGy?node=4042949045
+date: 2018-10-25
+startTime: '19:00'
+endTime: '21:00'
+---
```
Comment (benhylau) [inline _posts/2018-10-25-meet-n-greet.md:11]: We should tag this to all our events:

>This event has a [code of conduct](/code-of-conduct/).

File: _posts/2018-11-22-planning-session.md (hunk 1 above)
Comment (dasanchez) [reply _posts/2018-11-22-planning-session.md]: Whoops- fixed.

File: _posts/2018-10-25-meet-n-greet.md (hunk 2 above)
Comment (dasanchez) [reply _posts/2018-10-25-meet-n-greet.md:11]: All events are getting the Code of Conduct line now.

**Review by darkdrgn2k — APPROVED** [review]

**Review by benhylau — APPROVED** [review]

File: _posts/2018-11-22-planning-session.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+slug: planning-session
+text: 'Winter planning session for Toronto Mesh'
+location: HackLab.TO, 1266 Queen St
+locationLink: https://www.openstreetmap.org/node/3522641496
+date: 2018-11-22
+startTime: '18:30'
+endTime: '21:30'
+---
+
+This us Toronto Mesh's seventh planning session. We will regroup and plan for the winter and beyond! During this session we will:
```
Comment (garrying) [inline _posts/2018-11-22-planning-session.md]: `This us` -> `This is`?

File: _posts/2018-11-22-planning-session.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- Review all projects and efforts undertaken since April.
+- Re-evaluate our long-term goals.
+- Set the priorities and projects to work on for the next six months.
+- Assign working group leads for the next six months. You can see our current [working groups and governance processes here](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md)!
+
+We will meet at {{page.startTime | date:"%-I:%M %P "}} in [{{ page.location }}]({{ page.locationLink }}).
+
+This event has a [code of conduct](/code-of-conduct/).
+
+New to Toronto Mesh?
```
Comment (garrying) [inline _posts/2018-11-22-planning-session.md]: Maybe this could be a heading or bolded?

**Review by garrying — CHANGES_REQUESTED** [review]

File: _posts/2018-11-22-planning-session.md (hunk 3 above)
Comment (benhylau) [reply _posts/2018-11-22-planning-session.md]: Good catch

File: _posts/2018-11-22-planning-session.md (hunk 4 above)
Comment (dasanchez) [reply _posts/2018-11-22-planning-session.md]: Agreed! I'll update it.

**Comment by dasanchez** [comment]
Thanks for the comments @benhylau and @garrying! I updated the post accordingly.

**Review by garrying — APPROVED** [review]

**Pull request merged**